dist/js/materialize.js

```diff
@@ -220,8 +217,15 @@ jQuery.extend( jQuery.easing,
-    if (typeof define === 'function' && define.amd) {
-        define(['jquery', 'hammerjs'], factory);
+    if (typeof define === 'function' && define.amd == false) {
+        // hammerjs will already be publicly available
+        define(['jquery'], function($, Hammer) {
```

Comment:
Btw, the `Hammer` argument would overwrite the global `Hammer` variable, wouldn't it?
Thus, even though you are relying on the built-in Hammer implementation you will get an error in this function.